code-for-canada/project-thundercat
# [How To Page] Bug Fixes and Visual Improvements

# Description

The main goal of this Pull Request is to fix all known bugs in How To Page, but also to improve the visual. This PR contains the following changes:

- Updated section/subsection titles colors
- Made the subsection titles < h3 > instead of < p >
- Added French images and added a condition that allows the translation of the images when switching between English and French
- Created a js function that allows us to go back on the top of the page after selecting 'Next' and 'Back'
- Created a generic component called 'backTopTop' that uses the previous js function and added this generic component in the progressPane component, so we can click on 'Back on top' anytime
- Added some visual effects to 'Back to top' button, meaning that at the top of the page, the button is hidden and once you scroll down, it becomes visible

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Code cleanliness or refactor

## Screenshot

**New section/subsection titles colors/styles:**
![image](https://user-images.githubusercontent.com/23021242/53350498-2cb56200-38ed-11e9-944f-c232ada2da97.png)

**'Back to top' button hidden at the top of the page:**
![image](https://user-images.githubusercontent.com/23021242/53350587-5ff7f100-38ed-11e9-823c-10c82e5c295a.png)

**'Back to top' button becomes visible when we scroll down:**
![image](https://user-images.githubusercontent.com/23021242/53350635-7dc55600-38ed-11e9-9345-e9f0ff1d3346.png)

**English version of the images:**
![image](https://user-images.githubusercontent.com/23021242/53350725-b5340280-38ed-11e9-8190-f228d443abfb.png)

**French version of the images:**
![image](https://user-images.githubusercontent.com/23021242/53350764-c7ae3c00-38ed-11e9-875e-5e519a9daf69.png)


# Testing

Manual steps to reproduce this functionality:

1. Go to Prototype tab.
2. Click _Start eMIB Sample Test_.
3. Make sure that all the components look good, that the back to top button is working and that the translation is also working everywhere, even for the images.

Screenshot of unit tests run passing:

![image](https://user-images.githubusercontent.com/23021242/53357628-53c76000-38fc-11e9-907f-7b7f6f75e8de.png)

# Checklist
